www/artists/dhall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Why are you creating a page for Don Hall in a pull request to fix Iris Hause's page? This is irrelevant and should not be here. At best, it should be in a different pull request; at worst, it should not exist at all.